### PR TITLE
Remove redundant team ownership related code

### PR DESCRIPTION
**Pull request by MinetestSam**

### CODE REVIEW

**Comment by ClobberXD** [comment]
This PR only seems to remove the chat-command. What about the actual underlying systems that manage team-ownership?

**Comment by MinetestSam** [comment]
I haven't gone through the code to find instances where this is defined but if you want I can work on that.

**Comment by ClobberXD** [comment]
> if you want I can work on that.

Yes, that'd be great. I'm referring to stuff like the `auth` field in a `ctf.player` table that stores whether or not a player is a team-owner. I can't remember anything else from the top of my mind, but I'm sure there's a lot more related bloat in the codebase.

File: mods/ctf/ctf_chat/init.lua [hunk 1]
```diff
@@ -102,10 +102,7 @@ minetest.register_chatcommand("team", {
 			local count = 0
 			for _, value in pairs(ctf.team(param).players) do
 				count = count + 1
-				if value.auth then
-					minetest.chat_send_player(name, count .. ">> " .. value.name
-							.. " (team owner)")
-				else
+				if value then
```
Comment (ClobberXD) [inline mods/ctf/ctf_chat/init.lua]: This check is redundant, because `pairs` already iterates only over non-`nil` values in a table.

**Comment by ClobberXD** [comment]
Result of the latest commit (1f8003c):

screenshot_20191125_122010

Note the following:

- Coloured team and player names
- Team strength within parenthesis next to the team name
- Numerically listed player names (indented by two spaces)

**Pull request merged**